### DISCUSSION TITLE
S/Cloud RE

### DISCUSSION THREAD

**Discussion by msarmadi** [WeAreMahsaAmini/Core#136] [Introductions]
Grumpy, old, experienced and can contribute in following areas:

1. SRE/CRE, Python
2. System Design / Solution Architecture
3. Product Design and Strategy
4. Eng. Manager, Agile Methods
5. History in Business Development, Co-founder (IT services)
6. Documentation

**Comment by kaveh6** [comment]
please join discord server if you haven't already : https://discord.gg/ZygYz7vQ

**Reply by msarmadi** [reply]
affirmative
